src/aiida/storage/sqlite_dos/migrations/env.py

```diff
@@ -0,0 +1,54 @@
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Upper level SQLAlchemy migration functions."""
+
+from alembic import context
+
+
+def run_migrations_online():
+    """Run migrations in 'online' mode.
+
+    The connection should have been passed to the config, which we use to configue the migration context.
+    """
+    from aiida.storage.sqlite_zip.models import SqliteBase
```

Comment:
We really should extract SqliteBase and related out of sqlite_zip to something like `sqlite_common` or `sqlite_base` at some point.